### DISCUSSION TITLE
datax doriswriter插件文档部分sql语句错误

### DISCUSSION THREAD

**Discussion by junqiangChi** [apache/doris#16986] [RFC]
生态扩展->DataX doriswriter->示例->2.Doris表结构
其中的建表语句有语法错误
[问题定位点](https://doris.apache.org/zh-CN/docs/dev/ecosystem/datax#:~:text=%3D%27%27%3B-,2.Doris%E8%A1%A8%E7%BB%93%E6%9E%84,-CREATE%20TABLE%20%60)
![12312123](https://user-images.githubusercontent.com/75467791/220286112-086a7686-a601-4e7f-8060-de025b1dd3ba.png)

**Comment by zy-kkk** [comment]
可以来个PR修改一下
